packages/DataTable/src/DataTable.tsx

```diff
@@ -72,29 +70,36 @@ export interface DataTableProps {
   /** Override the table Column configuration for some particular rows */
   renderRow?: ((row: Row<TableDataItemType>) => React.ReactNode) | null;
 
+  /** If the table should be resized when the viewport size changed */
+  shouldResizeWithViewport?: boolean;
+
   /** Experimental prop */
   extraCellProps?: Record<string, any>;
 
   [x: string]: any;
 }
 
-const DataTable: React.FC<DataTableProps> & DataTableComposition = ({
-  a11yText,
-  children = null,
-  columns,
-  data,
-  height,
-  width,
-  borderType = gridTypes.HORIZONTAL,
-  getRowHeight = null,
-  hasZebraStripes = false,
-  isHeaderSticky = true,
-  renderRow = null,
-  extraCellProps = {},
-  ...moreProps
-}: DataTableProps) => {
+function DataTable(
+  {
+    a11yText,
+    children = null,
+    columns,
+    data,
+    height = "100vh",
+    width = "100vw",
```

Comment:
What would happen if these defaults were `null`? Sometimes we might actually want a table that has no max height (although that would make virtualization impossible).  

What about a max width of 100%? Would that be more sensible?

Reply:
Can they use `999999999px` instead? 😂  I don't really want to set the defaults to `null` cause it might break the infinite loader as well
However, if we really want, I think technically we can do it

> What about a max width of 100%?

Again `100%` is not currently supported, we need to calculate the value from the parent DOM element, I think starting from vh and vw is more feasible and we can collect some feedback at the same time

Reply:
Okay, let's leave the `100v` defaults for now.

Reply:
Ticket created: https://aclgrc.atlassian.net/browse/UXD-1967